src/main/java/thrift/logic/parser/CliSyntax.java

```diff
@@ -13,4 +13,6 @@
     public static final Prefix PREFIX_REMARK = new Prefix("r/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_VALUE = new Prefix("v/");
+    public static final Prefix PREFIX_CURRENCY = new Prefix("c/");
```

Comment:
Maybe we should keep this list sorted in alphabetical form?

Reply:
Sorted!